### PR TITLE
Create kubelog.yaml, Add Kubelog plugin

### PR DESCRIPTION
**Pull request by jfvilas**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/kubelog.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Kubelog
+author: Julio Fernandez
+authorUrl: https://github.com/jfvilas
+category: Observability
+description: Kubelog provides your Backstage users the ability to view logs of your Kubernetes objects linked to Backstage entities. Security controls can be applied.
+documentation: https://github.com/jfvilas/kubelog
+iconUrl: https://raw.githubusercontent.com/jfvilas/kubelog/master/src/assets/kubelog-logo.png
+npmPackageName: '@jfvilas/plugin-kubelog'
```
Comment (awanlin) [inline microsite/data/plugins/kubelog.yaml:9]: You package still points to the other account, here:

https://github.com/jfvilas/kubelog/blob/f462bd0f54c1cb07320916ca0c5dd4c4e9adab17/package.json#L24

And also here:

https://github.com/jfvilas/kubelog/blob/f462bd0f54c1cb07320916ca0c5dd4c4e9adab17/package.json#L32

Can you correct those please?

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for the contribution, this looks like a helpful plugin, I left one comment @jfvilas 

Generally speaking please don't close the PR but make the requested changes and update the PR as needed. This makes it easy for you and reviewers to keep track of things 👍

**Comment by jfvilas** [comment]
Hi @awanlin , I've closed the PR in order to have this new PR associated to my new username, sorry for the inconvenience. I've already changed the package.json references to new repository, pushed everything a publish a new version of the NPM packages. Thanks.

**Review by awanlin — APPROVED** [review]
Awesome! 🚀

**Comment by awanlin** [comment]
@freben @camilaibs this is good to go, just need your review 👍

**Review by freben — APPROVED** [review]

**Pull request merged**